internal/operators/ocs/validations.go

```diff
@@ -165,24 +167,21 @@ func (o *ocsValidator) nodeResources(host *models.Host, cpuCount *int64, totalRA
 		return err
 	}
 
-	disks, err := o.hostApi.GetHostValidDisks(host)
-	if err != nil {
-		o.log.Error("failed to get valid disks due to ", err.Error())
-		return err
-	}
+	o.log.Info("Counting disks with driveType HDD or SSD")
+	disks := getDisks(inventory.Disks)
 
-	if len(disks) > 1 { // OCS must use the non-boot disks
-		diskCPU := int64((len(disks) - 1)) * o.OCSRequiredDiskCPUCount
-		diskRAM := int64((len(disks) - 1)) * o.OCSRequiredDiskRAMGB
+	if disks > 1 { // OCS must use the non-boot disks
+		diskCPU := int64(disks-1) * o.OCSRequiredDiskCPUCount
```

Comment:
Perhaps better to rename these requiredOSDCPU and requiredOSDRAM

Reply:
fixed